.github/workflows/deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - "main"
+      - "feature/deployment"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 18.x ]
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: npm install
```

Comment:
Might be useful to add actions/setup-node to cache npm packages and speed up build time: https://github.com/actions/setup-node#matrix-testing